### PR TITLE
add get property endpoint

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: property-api/V1/UseCase/GetPropertyUseCase.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        public class GetPropertyByRefResponse
+        {
+            public readonly bool Success;
+            public readonly Property Property;
+
+            public GetPropertyByRefResponse(Property property)
+            {
+                if (property != null)
+                {
+                    Success = true;
```
Comment (LBHmsaitz) [inline property-api/V1/UseCase/GetPropertyUseCase.cs]: Nicer than nulls!

File: property-api.Tests/property-api.Tests.csproj [hunk 2]
```diff
@@ -13,10 +13,17 @@
     <PackageReference Include="Bogus" Version="25.0.4" />
     <PackageReference Include="Moq" Version="4.10.1" />
     <PackageReference Include="nunit" Version="3.11.0" />
+    <PackageReference Include="Tynamix.ObjectFiller" Version="1.5.5" />
```
Comment (LBHmsaitz) [inline property-api.Tests/property-api.Tests.csproj]: Not necessary anymore

File: property-api/Startup.cs [hunk 3]
```diff
@@ -25,34 +27,37 @@ public Startup(IConfiguration configuration)
...
+            var mappingConfig = new MapperConfiguration(cfg => { cfg.CreateMap<UhProperty, Property>(); });
+            var propertyFactory = new PropertyFactory(mappingConfig.CreateMapper());
+            services.AddSingleton(propertyFactory);
+        }
 
+        private static void ConfigureDbContext(IServiceCollection services)
+        {
             DbContextOptionsBuilder builder = new DbContextOptionsBuilder()
-                .UseSqlServer(connectionString);
+                .UseSqlServer("Data Source=localhost;Initial Catalog=uhsimulator;user id=sa;password=Rooty-Tooty;");
```
Comment (LBHSPreston) [inline property-api/Startup.cs]: Hard coded connection string to test db.

File: property-api/V1/Controllers/PropertyController.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            _getPropertyUseCase = getPropertyUseCase;
+            _logger = logger;
+        }
+
+        [HttpGet("{propertyReference}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(GetPropertyUseCase.GetPropertyByRefResponse), 200)]
+        [ProducesResponseType(typeof(NotFoundResult), 404)]
+        public IActionResult GetByReference(string propertyReference)
```
Comment (LBHSPreston) [inline property-api/V1/Controllers/PropertyController.cs]: Should be a URL parameter.

Comment (cob16) [reply property-api/V1/Controllers/PropertyController.cs]: fixed

File: property-api/Startup.cs (hunk 3 above)
Comment (cob16) [reply property-api/Startup.cs]: fixed

**Pull request merged**